enhancements/cnf-tests-split.md

```diff
@@ -0,0 +1,204 @@
+---
+title: cnf-tests-split
+authors:
+  - "@liornoy"
+reviewers:
+  - "@cnf-network-team"
+  - "@cnf-ci-team"
+  - "@cnf-compute-team"
+approvers:
+  - "@cnf-network-team"
+  - "@cnf-ci-team"
+  - "@cnf-compute-team"
+creation-date: 2023-06-01
+last-updated: 2023-06-01
+tracking-link:
+  - https://issues.redhat.com/browse/CNF-7583
+---
+
+# CNF-tests split
+
+The cnf-tests is a sub-project in the cnf-features-deploy repository, aimed to run tests against various CNF workloads on top of an Openshift cluster.  
```

Comment:
This is not exactly true. The cnf tests suite aims to validate the various openshift features necessary to run cnf workloads on a cluster.